xonsh/environ.py

```diff
@@ -710,7 +711,7 @@ def locate_binary(name):
 
     try:
         return next(chain.from_iterable(yield_executables(directory, name) for
-                    directory in builtins.__xonsh_env__.get('PATH') if os.path.isdir(directory)))
+                    directory in ([_get_cwd()] + builtins.__xonsh_env__.get('PATH')) if os.path.isdir(directory)))
```

Comment:
@scopatz - I'm not sure if that's a bug -- at least on *NIX `os.path` will use `posixpath.py` which explicity defines the current directory as `.`